geode-management/src/main/java/org/apache/geode/management/client/ClusterManagementServiceBuilder.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.client;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLContext;
+
+import org.springframework.http.client.ClientHttpRequestFactory;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.management.api.CacheMarker;
+import org.apache.geode.management.api.ClusterManagementService;
+
+@Experimental
+public class ClusterManagementServiceBuilder {
+
+  public static JavaBuilder buildWithUrl() {
```

Comment:
Since we're not actually using a URL, can this be more specific like `buildWithEndpoint` or `buildWithAddress` - and then also change the corresponding `set*` method.